nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/io/BulkJsonWriter.java

```diff
@@ -50,7 +50,7 @@ public BulkJsonWriter() {
 
```

Comment:
Can we rename this class `BulkStatusJsonWriter` ?

Reply:
see above about io refactoring